### PR TITLE
Story/sus 13

### PR DESCRIPTION
**Pull request by namrathaov**
[SUS-13] Added alias for concept name in edit configuration page and search page.

### CODE REVIEW

File: sustainability/src/main/java/edu/asu/diging/sustainability/core/service/impl/ConceptManager.java [hunk 1]
```diff
@@ -66,4 +66,18 @@ public void storeConceptSearchCategories(String conceptId, List<SearchCategory>
...
     }
+
+    /**
+     * Store the alias for concept name for the concept with given Id.
+     * 
+     * @param conceptId The id of the concept for which the alias is provided.
+     * @param alias Alias for the concept name to be saved.
+     */
+    @Override
+    public void storeConceptAlias(String conceptId, String alias) {
```
Comment (jdamerow) [inline sustainability/src/main/java/edu/asu/diging/sustainability/core/service/impl/ConceptManager.java]: merge with 'storeConceptSearchCategories'. no reason there would be an extra method for bot properties if there could just be one update method.

Comment (jdamerow) [reply sustainability/src/main/java/edu/asu/diging/sustainability/core/service/impl/ConceptManager.java]: also, let's write some tests at least for the update method.

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by namrathaov** [comment]
Merged search categories and alias in one function. 
Added unit test for update function in Concept manager.

File: sustainability/src/test/java/edu/asu/diging/sustainability/core/service/impl/ConceptManagerTest.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    }
+
+    @Test
+    public void test_storeConceptSearchCategoriesAndAlias_success() {
+        concept.setSearchCategories(categoryList);
+        concept.setAlias(alias);
+        Mockito.when(conceptRepo.findById("ConceptId1")).thenReturn(Optional.of(concept));
+        Mockito.when(conceptRepo.save(concept)).thenReturn(concept);
+        conceptManager.storeConceptSearchCategoriesAndAlias(concept.getId(), categoryList, alias);
+        verify(conceptRepo).save(concept);
```
Comment (jdamerow) [inline sustainability/src/test/java/edu/asu/diging/sustainability/core/service/impl/ConceptManagerTest.java:44]: you probably want to make sure category list and alias are updated

File: sustainability/src/test/java/edu/asu/diging/sustainability/core/service/impl/ConceptManagerTest.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+    private List<SearchCategory> categoryList = new ArrayList<SearchCategory>();
+
+    @Mock
+    private ConceptRepository conceptRepo;
+
+    @InjectMocks
+    private ConceptManager conceptManager = new ConceptManager();
+
+    private Concept concept = new Concept();
```
Comment (jdamerow) [inline sustainability/src/test/java/edu/asu/diging/sustainability/core/service/impl/ConceptManagerTest.java]: no reason to make concept and categoryList class variables

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by namrathaov** [comment]
Updated test case to verify search category and alias are updated.

**Comment by jdamerow** [comment]
ok to test

**Review by jdamerow — APPROVED** [review]

**Pull request merged**